test/modules/standard/Spawn/spawn-popen-input-output-long.chpl

```diff
@@ -0,0 +1,26 @@
+use Spawn;
+
+var sub = spawn(["cat"], stdin=PIPE, stdout=PIPE, stderr=FORWARD);
+
+config const n = 10000;
+for i in 1..n {
+  //writeln("writing line: ", i);
```

Comment:
Still have a commented out line here, do you want to treat it the same as the error case writes or remove it entirely?